roles/etcd/tasks/system_container.yml

```diff
@@ -24,23 +24,30 @@
   systemd:
     name: etcd
     state: stopped
-    enabled: yes
+    enabled: no
     masked: no
     daemon_reload: yes
   register: task_result
   failed_when: task_result|failed and 'could not' not in task_result.msg|lower
-  when: "'etcd' in etcd_result.stdout"
+  when: "'etcd' not in etcd_result.stdout"
 
 - name: Disable etcd_container
   systemd:
     name: etcd_container
     state: stopped
     enabled: no
-    masked: yes
     daemon_reload: yes
   register: task_result
   failed_when: task_result|failed and 'could not' not in task_result.msg|lower
 
+- name: Remove etcd_container.service
+  file:
+    path: /etc/systemd/system/etcd_container.service
+    state: absent
+
+- name: Systemd reload configuration
+  systemd: name=etcd_container daemon_reload=yes
```

Comment:
I don't want to nak this becase you'd have to go back through the ci jobs, but lets make sure that we use the same format that in the future we use the multi-line format.
```
- name: Systemd reload configuration
  systemd:
    name: etcd_container
    daemon_reload: yes
```